src/Flicking.ts

```diff
@@ -649,10 +661,10 @@ class Flicking extends Component<FlickingEvent> {
     let type: MoveTypeStringOption | undefined;
     let moveTypeOptions: MoveTypeObjectOption | undefined;
 
-    if (isString(moveType) && moveType in DEFAULT_MOVE_TYPE_OPTIONS) {
+    if (typeof moveType === "string" && moveType in DEFAULT_MOVE_TYPE_OPTIONS) {
```

Comment:
I think we can solve the duplicate code, but is there a reason to remove isString?

Reply:
That's right, I think I've removed that because once while I'm working on my code there was a moment that `=== "string"` is used only once.
I'll revert that, thanks for checking :)